spec/models/content_item_spec.rb

```diff
@@ -11,12 +11,16 @@
   end
 
   describe "#create_or_update" do
-    let(:content_item_1) { { content_id: "first_id" } }
-    let(:content_item_2) { { content_id: "second_id" } }
+    let(:content_item_1) { { content_id: "first_id", taxons: [] } }
```

Comment:
@theleebriggs I would try to be much more explicit at the test level. There are too many `let` and creations in the before context that are global to all instances.

What are your thoughts about setting up the fixture per test?